pilot/pkg/serviceregistry/kube/controller/controller.go

```diff
@@ -109,6 +109,7 @@ type Options struct {
 	// MeshServiceController is a mesh-wide service Controller.
 	MeshServiceController *aggregate.Controller
 
+	ResyncPeriod time.Duration
```

Comment:
This looks unrelated to the logKubernetesApiRequests thing.

Reply:
Yeah, I couldn't really find this being used anywhere. I guess we can remove it?

Reply:
For whatever reason, this comes from https://github.com/maistra/istio/pull/585